src/mui/list/DatagridBody.js

```diff
@@ -47,7 +48,7 @@ const DatagridBody = ({
                                         index === 0 ? (
                                             styles.cell['td:first-child']
                                         ) : (
-                                            styles.cell.td
+                                            index === children.length - 1 ? styles.cell['td:last-child'] : styles.cell.td
```

Comment:
As it becomes a more complicated operation, I think it could go in a function, or at least it could be nice to keep the aor coding style with parenthesis and line break for clarity.

Reply:
ok